tldw_Server_API/app/services/media_ingest_jobs_worker.py

```
@@ -164,15 +166,26 @@ async def _schedule_embeddings(
             chunk_overlap=getattr(form_data, "overlap", None) or 200,
+            user_id=user_id,
         )
+        allow_zero = bool(result.get("allow_zero_embeddings"))
+        if result.get("status") == "success" or allow_zero:
+            mark_media_as_processed(db_instance=db, media_id=media_id)
+        else:
+            db.mark_embeddings_error(
+                media_id,
+                str(result.get("error") or result.get("message") or "Embedding generation failed"),
+            )
     except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)